index.html

```diff
@@ -4,15 +4,16 @@
   <head>
     <title>Lambda School Newsfeed</title>
     <link rel="stylesheet" type="text/css" media="screen" href="./CSS/index.css" />
-    <script src="./Menu/Menu.js"></script>
-    <script src="./Article/Article.js"></script>
+    
   </head>
   <body>
     <div class="header">
       <img class="menu-button" src="./assets/menu.png"/>
       <h1>Lambda School Newsfeed</h1>
     </div>
     <div class="articles"></div>
+    <script src="./Menu/Menu.js"></script>
+    <script src="./Article/Article.js"></script>
```

Comment:
I would suggest keeping your script tags within the Head and marking them as 'async' as default unless you have a specific use case for 'defer' which is the same as putting the JS script at the bottom of the page.